src/NCHttpDialog.cc

```diff
@@ -164,6 +164,10 @@ wint_t NCHttpDialog::getch( int timeout_millisec )
     {
 	   // wait for input (block)
 	   wait_for_input(timeout_millisec);
+        // In case of Multi and Custom ItemSelector ncurses UI was not handling events properly. 
```

Comment:
I don't quite understand this. Do we need to change event handling for the NCItemSelector widgets?

Reply:
I've asked @lslezak about it, so in case of using wrapper from ruby code, everything works fine, but in case I use plain C++ examples with `notify` being enabled, events get stuck, in the logs there is `evt::btn` and rest api hangs unless I do something in the UI manually. And the only problematic control was ItemSelector. This is the solution I managed to find, but in this particular case: 
![image](https://user-images.githubusercontent.com/28305643/70155833-445b1b80-16b3-11ea-99e0-24810eaccea7.png)

Reply:
As it works with ruby wrapper, we agreed to leave code as it is, so the change is reverted.